taxcalc/tests/test_policy.py

```diff
@@ -699,26 +699,18 @@ def generate_section_dictionary(html_text):
             'Additional Medicare FICA': 0
         },
         'Social Security Taxability': {
```

Comment:
@Peter-Metz, could you help me understand what is going on in test_policy.py? 

https://github.com/PSLmodels/Tax-Calculator/pull/2372/files#diff-dc74f969d76006abbdf3bcc5b95240e4L702-L721

Reply:
@MattHJensen sure thing. The gist is that `test_policy` creates a dictionary called `valid_dict` and compares it to `policy_current_law` to check `section_1` and `section_2` titles. For example:

```python
valid_dict = {
       # corresponds to PCL section_1
        'Payroll Taxes': {
            # corresponds to PCL section_2
            'Social Security FICA': 0
        }
```

Since this PR removes some of the `section_2` titles, we modify `valid_dict` with:

```python
valid_dict = {
        'Payroll Taxes': {
            '': 0
        }
```

Does that clear things up?

Reply:
Thanks, @Peter-Metz